docs/sts/rw-ldap-username.json

```diff
@@ -4,10 +4,10 @@
     {
       "Effect": "Allow",
       "Action": [
-        "s3:ListBucket"
+        "s3:*"
       ],
       "Resource": [
-        "arn:aws:s3:::${ldap:user}"
+        "arn:aws:s3:::${ldap:username}/*"
```

Comment:
Oh nice, I didn't know this is possible, do we have a similar thing for regular IAM users ?

Reply:
yes its called `aws:username` @vadmeste and for jwt `jwt:preferred_username` ... `jwt:email`